### DISCUSSION TITLE
Show breakes of Pretalx

### DISCUSSION THREAD

**Discussion by navlisilvan** [EventFahrplan/EventFahrplan#462] [Ideas]
Hi,

i was wondering if it would be a good feature to show the breakes of Pretalx in the EventFahrplan. 

what do you think?

**Comment by johnjohndoe** [comment]
Hi. If I understand you correctly you would like to fill in session slots which are not talks or workshops but rather breaks such as lunch, tea time, ...

Currently, those are mostly shown as space (where the background shines through). It is however up to the program folks to insert sessions for breaks at any time. The app will handle them automatically and render the same way as any other session.

It makes sense to communicate this early enough between program folks and app developers because those breaks should be visually separated from "normal" sessions of an event.

Do you relate to a specific event where the app is used?

I hope this answers you question. If not, feel free to continue here.

**Reply by navlisilvan** [reply]
We are testing Pretalx and Eventfahrplan with this event:
https://workshops.acrolotl.de/al22/schedule/#

There we used the Pretalx-"Breakes"-Funktion but is saw that they are not in the XML-Export. :/
So an seperat event for the breakes would be the solution. Thx..
